dspace-oai/src/main/java/org/dspace/xoai/app/XOAI.java

```diff
@@ -757,14 +745,9 @@ public void indexItems(java.util.Collection<Item> items) throws Exception {
                 deleteItemByQuery(item);
                 solrServerResolver.getServer().add(this.index(item));
             } catch (IOException | XMLStreamException | SQLException | WritingXmlException | SolrServerException e) {
-                // Do not throw RuntimeException in tests
-                if (this.isTest()) {
```

Comment:
Why?

Reply:
Something is wrong because, in this branch, this code is already not present.